libs/dh/core/shell/src/lib/dh-core-shell.routes.ts

```diff
@@ -32,7 +37,7 @@ export const dhCoreShellRoutes: Routes = [
     children: [
       {
         path: '',
-        redirectTo: getPath<BasePaths>('message-archive'),
+        redirectTo: `${getPath<BasePaths>('metering-point')}/${getPath<MeteringPointSubPaths>('search')}`,
```

Comment:
suggestion: what about creating an async redirect here which can decide where to send the user? That way there will be no need to extend the `PermissionGuard` with the extra param.